frontend/model/contracts/group.js

```diff
@@ -342,7 +342,19 @@ const leaveChatRoomAction = (state, chatRoomID, memberID, actorID, leavingGroup)
   sbp('gi.actions/chatroom/leave', {
     contractID: chatRoomID,
     data: sendingData,
-    ...extraParams
+    ...extraParams,
+    hooks: {
+      onprocessed: () => {
+        const rootState = sbp('state/vuex/state')
+        if (memberID === rootState.loggedIn.identityContractID) {
+          sbp('chelonia/queueInvocation', chatRoomID, () => {
+            sbp('chelonia/contract/release', chatRoomID).catch(e => {
+              console.error(`[leaveChatRoomAction] Error releasing chatroom ${chatRoomID}`, e)
+            })
```

Comment:
Could you add a comment above line 350 to explain why this is being pushed to a queue?

Reply:
Okay, sure.